src/common/utils/registry/repository.go

```diff
@@ -99,13 +101,6 @@ func (r *Repository) ListTag() ([]string, error) {
 		sort.Strings(tags)
 		tags = tagsResp.Tags
 
-		return tags, nil
-	} else if resp.StatusCode == http.StatusNotFound {
-
```

Comment:
The issue of distribution is not fixed yet, why we remove the workaround here?

Reply:
This workaround will hide error message when it is true. Can we 
just hidden error in the UI to fix that's issue of harbor.

Reply:
No. The method is used by other functions and it's a bug of registry API, we should not expose it to upper callers

Reply:
👌, I have reverted.